src/AgGrid.fs

```diff
@@ -63,7 +61,7 @@ type ColumnDef<'row, 'value> =
     static member inline autoComparator = columnDefProp<'row, 'value> ("comparator" ==> compare)
     static member inline cellClass (setClass:'value -> 'row -> #seq<string>) = columnDefProp<'row, 'value> ("cellClass" ==> fun p -> setClass p?value p?data |> Seq.toArray)
     static member inline cellClassRules (rules: (string*('value -> 'row -> bool)) list) = columnDefProp<'row, 'value> ("cellClassRules" ==> (rules |> List.map (fun (className, rule) -> className ==> fun p -> rule p?value p?data) |> createObj))
-    static member cellRendererFramework (render:'value -> 'row -> ReactElement) = columnDefProp<'row, 'value> ("cellRendererFramework" ==> fun p -> CellRendererComponent(render, p))
+    static member cellRendererFramework (render:'value -> 'row -> ReactElement) = columnDefProp<'row, 'value> ("cellRenderer" ==> fun p -> CellRendererComponent(render, p))

```

Comment:
This package's API is intended to the npm package's API. If we're using a JS property with a different name, we should change the name of the F# function too.

```suggestion
    static member cellRenderer (render:'value -> 'row -> ReactElement) = columnDefProp<'row, 'value> ("cellRenderer" ==> fun p -> CellRendererComponent(render, p))

```

Perhaps worth looking at deprecated/obsolete attributes that we could add to the old function to provide users with some guidance when upgrading. Something like this (can't remember exact syntax)

```fsharp
    [<Obsolete("cellRendererFramework isn't supported in the latest version of AgGrid. Use cellRenderer instead.", true)>]
    static member cellRendererFramework _ = failwith cellRendererFramework isn't supported in the latest version of AgGrid. Use cellRenderer instead."

```

In addition, I vaguely remember cellRenderer and cellRendererFramework having different behaviours. Can you spend a short amount of time investigating to make sure that we're not accidentally changing behaviour? If you don't make any headway in 15 minutes, let me know and we can decide what to do at that point.

Reply:
Yes good shout with the obsolete change. I have looked into the cellRenderer and it is a more enhanced version of cellRendererFramework from what I can see.